ddl/ddl.go

```diff
@@ -1749,7 +1749,8 @@ type session struct {
 	sessionctx.Context
 }
 
-func newSession(s sessionctx.Context) *session {
+// NewSession news the session and it is export for testing.
+func NewSession(s sessionctx.Context) *session {
```

Comment:
You can add the following code to `ddl/export_test.go` so that it doesn't have to be exported.

```
var NewSession = newSession
```

Reply:
The other way is to use ```[internal](https://go.dev/doc/go1.4#internalpackages)``` package.